### PR TITLE
Added timeout option to authentication module

### PR DESCRIPTION
**Pull request by mrendo**
### Desired Outcome

We noticed now we have turned on two factor we have a problem with the module timing out before we have pressed the push notification.

### Implemented Changes

*Describe how the desired outcome above has been achieved with this PR. In
particular, consider:*

Adding a timeout option to the authentication module, tested it using a local copy of the module with my Cyberark system

### Connected Issue/Story

Resolves #[relevant GitHub issue(s), e.g. 76]  #47

CyberArk internal issue link: [insert issue ID]()

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace: "cyberark"
 name: "pas"
-version: "1.0.14"
+version: "1.0.15"
```
Comment (infamousjoeg) [inline galaxy.yml:3]: Please bump this to 1.0.16

**Review by infamousjoeg — CHANGES_REQUESTED** [review]
After bumping version to 1.0.16 this looks good to me and can be merged.

File: galaxy.yml (hunk 1 above)
Comment (mrendo) [reply galaxy.yml]: All done I believe thanks @infamousjoeg

**Review by infamousjoeg — APPROVED** [review]

**Comment by infamousjoeg** [comment]
Will merge and publish to Galaxy in the morning.

**Pull request merged**